SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -328,10 +331,13 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	Sender->Init(this, &TimerManager);
 	Receiver->Init(this, &TimerManager);
 	GlobalStateManager->Init(this, &TimerManager);
-	VirtualWorkerTranslator->Init(this);
-	// TODO(zoning): This currently hard codes the desired number of virtual workers. This should be retrieved
-	// from the configuration.
-	VirtualWorkerTranslator->SetDesiredVirtualWorkerCount(2);
+	if (GetDefault<USpatialGDKSettings>()->bEnableUnrealLoadBalancer)
+	{
+		VirtualWorkerTranslator->Init(this);
+		// TODO(zoning): This currently hard codes the desired number of virtual workers. This should be retrieved
+		// from the configuration.
+		VirtualWorkerTranslator->SetDesiredVirtualWorkerCount(2);
```

Comment:
Nit: not related to this PR, but maybe we should have a jira ticket linked to this TODO?

Reply:
This has been re-worked in master.